GNUmakefile

```diff
@@ -49,7 +49,7 @@ PROTO_COMPARE_TAG ?= v1.0.3$(if $(findstring ent,$(GO_TAGS)),+ent,)
 
 # LAST_RELEASE is the git sha of the latest release corresponding to this branch. main should have the latest
 # published release, and release branches should point to the latest published release in the X.Y release line.
-LAST_RELEASE ?= v1.4.3
+LAST_RELEASE ?= v1.5.0
```

Comment:
Ugh...I think this caused this extra changelog entry (it was supposed to be `v1.4.4`):

```
SECURITY:
* build: Update to go1.20.1 [[GH-16182](https://github.com/hashicorp/nomad/issues/16182)]
```

We need to double-check our security release checklist to make sure it's updating this value properly.

Reply:
Ah I wondered where they came from. I pulled them out of the 1.4.5 and 1.3.10 entries. Feel free to merge when the checks pass, I am pretty much ready to check out now.